components/UI/ServiceSubmenu.js

```diff
@@ -4,12 +4,14 @@ import { useTranslation } from 'react-i18next';
 import _ from 'lodash';
 import { MenuAddIcon } from '@remixicons/react/line';
 
+import TextButton from './TextButton';
+
 const Header = styled.header`
   display: grid;
   grid-template-columns: 1fr auto;
 `;
 
-const ToggleButton = styled.button`
+const ToggleButton = styled(TextButton)`
```

Comment:
Now that we're using the `TextButton` as our base here, do we really still need to unset things like `background-color` and `border` in the lines below? (19 & 20)
Isn't the point of the `<TextButton>` component to do exactly _that_?